Sources/Conformance/conformance.pb.swift

```diff
@@ -245,18 +245,24 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf.Prot
 
   init() {}
 
-  public mutating func _protoc_generated_decodeField<T: SwiftProtobuf.FieldDecoder>(setter: inout T, protoFieldNumber: Int) throws {
-    switch protoFieldNumber {
-    case 1, 2: try payload.decodeField(setter: &setter, protoFieldNumber: protoFieldNumber)
-    case 3: try setter.decodeSingularField(fieldType: Conformance_WireFormat.self, value: &requestedOutputFormat)
+  public mutating func _protoc_generated_decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
```

Comment:
> ... we should just generate everything in the message and have it reach into the storage as needed to access the field properties...

This would require care to ensure that "reaching into the storage" did not pay too much for the copy-on-write and refcount overhead.  In particular, you don't want to go through the accessors for each separate field.